README.md

```diff
@@ -40,6 +40,12 @@ better queries which in turn will produce better results.
 
 To start developing open a terminal and run:
 
+```
+env --chdir=frontend nix develop -c yarn install
```

Comment:
do we need env to do something like `--chdir` here?

Reply:
yes - i just wondered if using `env` was overkill for that, but to be fair that was a nitpick :sweat_smile: